### ISSUE TITLE
[minor] search for search engine names

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [ueokande/vim-vixen#305]
### I'm opening this issue because:

- [ X ] I'll report a bug
- [ ] I'll propose a new feature

### Description
In the `:t` or :o` commands, say my default search engine is `qwant` and I want to search for *google*, I actually mean it : search for the name of the company *google*.  It will load the engine google with an empty keyword strings instead of giving me the `qwant` result for the keyword *google*.

This can also cause issue with some search engines that return 404 when you try to search for nothing. (I can provide examples if needed).

#### Steps to Reproduce
(another example is shown here)
1. set duckduckgo as your default search engine
2. set qwant as your second search engine
3. in vimvixen type `:o qwant` and instead of having a duckduckgo page about qwant you'll end up with loading an empty qwant page.

#### System configuration

- Operating system:Ubuntu 16.04
- Firefox version:57.0.4
- Vim-Vixen version:0.8

**Comment by AlexeySachkov** [comment]
Possible workaround (at least works for me): to search "google" when you have search engine named `google` you need to add quotes: type `:t "google"`

**Comment by thiswillbeyourgithub** [comment]
@AlexeySachkov that's right, one could also change his engine names in vimvixen to something slightly off like "goog" and this would not happen. But that's still a bug in my opinion.

And if someone thinks that it's the expected behavior and not a bug, please notice that you won't endup at (example with google) `https://google.com/` but at `https://google.com/search?q=` which for google is fine (it redirects on its own) but some other engines might return 404.
